docker-compose.validator.yml

```diff
@@ -11,7 +11,7 @@ services:
       - ./keystores:/keystores
       - ./secrets:/secrets
     env_file: .env
-    command: validator --rootDir /data --keystoresDir /keystores --secretsDir /secrets --server http://beacon_node:9596
+    command: validator --rootDir /data --keystoresDir /keystores --secretsDir /secrets --server http://beacon_node:9596 --network ${NETWORK}
```

Comment:
Does it work with `LODESTAR_NETWORK` on the .env file?

Reply:
No, I didn't have to use the prefix at all.  I just passed in NETWORK=pyrmont in the .env file.